From 9803754862997f6f6abbc78b19893ca6d6087c39 Mon Sep 17 00:00:00 2001
From: "Tan, Thuan.HuiX" <thuan.huix.tan@intel.com>
Date: Thu, 16 May 2019 11:09:46 +0800
Subject: [PATCH 2329/2345] media: intel-ipu4: ox03a10: Modify I2C bus ID and
 GPIO pin number

Fix TI960 can't detect the ox03a10 sensor in port 0&1

Change-Id: I5bde86ed17bdff347776e03df625f8addfd75e5f
Signed-off-by: Shen Tao <taox.shen@intel.com>
Signed-off-by: Khai Wen Ng <khai.wen.ng@intel.com>
---
 drivers/media/platform/intel/ipu4-bxt-p-pdata.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
index 0bd97a9..07976a4 100644
--- a/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
+++ b/drivers/media/platform/intel/ipu4-bxt-p-pdata.c
@@ -1094,7 +1094,7 @@ static struct crlmodule_platform_data ov495_pdata = {
 
 #if IS_ENABLED(CONFIG_VIDEO_TI960)
 #define TI960_I2C_ADAPTER	2
-#define TI960_I2C_ADAPTER_2	7
+#define TI960_I2C_ADAPTER_2	4
 #define TI960_LANES	4
 
 static struct ipu_isys_csi2_config ti960_csi2_cfg = {
@@ -1325,7 +1325,7 @@ static struct ti960_subdev_info ti960_subdevs_2[] = {
 static struct ti960_pdata ti960_pdata = {
 	.subdev_info = ti960_subdevs,
 	.subdev_num = ARRAY_SIZE(ti960_subdevs),
-	.reset_gpio = GPIO_BASE + 63,
+	.reset_gpio = GPIO_BASE + 62,
 	.suffix = 'a',
 };
 
@@ -1344,7 +1344,7 @@ static struct ipu_isys_subdev_info ti960_sd = {
 static struct ti960_pdata ti960_pdata_2 = {
 	.subdev_info = ti960_subdevs_2,
 	.subdev_num = ARRAY_SIZE(ti960_subdevs_2),
-	.reset_gpio = GPIO_BASE + 66,
+	.reset_gpio = GPIO_BASE + 69,
 	.suffix = 'b',
 };
 
-- 
2.7.4

